disnake/ui/_types.py

```diff
@@ -24,7 +27,9 @@
     from .select import ChannelSelect, MentionableSelect, RoleSelect, StringSelect, UserSelect
     from .view import View
 
-V_co = TypeVar("V_co", bound="Optional[View]", covariant=True)
+    V_co = TypeVar("V_co", bound="Optional[View]", covariant=True, default=Optional[View])
+else:
+    V_co = TypeVar("V_co", bound="Optional[View]", covariant=True)
```

Comment:
https://github.com/DisnakeDev/disnake/pull/1441/files#r2443495659

```suggestion
    V_co = TypeVar("V_co")
```